chainer/backends/cuda.py

```diff
@@ -535,7 +532,10 @@ def get_max_workspace_size():
         int: The workspace size for cuDNN.
 
     """
-    return _max_workspace_size
+    # To avoid error on no cuDNN environment
+    if cudnn_enabled:
+        return cudnn.get_max_workspace_size()
+    return 0
```

Comment:
This seems to change the behavior for non-cuDNN environment. Should we raise warning then this function is called in a such situation, or is the current behavior maybe acceptable?

Reply:
Previous functions did not raise exceptions in non-cuDNN environment.
I agree with the policy to change in the future. However, this PR holds the behavior of the old function.